java/arcs/jvm/host/ExplicitHostRegistry.kt

```diff
@@ -19,7 +19,7 @@ import arcs.core.host.api.Particle
  * explicitly registered [ArcHost]s and [Particle]s invoked by [HostRegistry.registerHost] and
  * [ExplicitHostRegistry.registerParticles].
  */
-class ExplicitHostRegistry : HostRegistry {
+class ExplicitHostRegistry : HostRegistry() {
```

Comment:
You also need to update AndroidManifestHostRegistry

Reply:
It's the first file in the pr. I don't think there is anything else needed in that file?